packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -358,6 +358,15 @@ const getStyles = (theme: GrafanaTheme2) => {
       display: 'flex',
       flexDirection: 'column',
 
+      '> *': {
+        zIndex: 0,
+      },
+
+      // matches .react-grid-item styles in _dashboard_grid.scss to ensure any contained tooltips occlude adjacent panels
+      '&:hover, &:active, &:focus': {
+        zIndex: 999,
+      },
```

Comment:
this handles the case then the panel is not in a grid, such as via PanelRenderer

Reply:
🤔 it seems this change conflicts with the ability to resize the panel using drag a drop. 

https://github.com/grafana/grafana/assets/239999/314d5432-7265-4adf-aad2-915fb6d7527d

Reply:
ah, good catch. i wonder what the z-index is of that drag handle :thinking:

Reply:
fixed in [b3dcbe8](https://github.com/grafana/grafana/pull/75597/commits/b3dcbe8ee05c82104df7028689567045b565bedd)